scripts/post.lua

```diff
@@ -1,17 +1,24 @@
 local pipelineDepth = 1
 
+function readData(filename)
+   local f = io.open(filename, "rb")
+   local data = f:read("*a")
+   f:close()
+   return data
+end
+
 function init(args)
    wrk.method = "POST"
-   wrk.body   = "foo=bar&baz=quux"
-   wrk.headers["Content-Type"] = "application/x-www-form-urlencoded"
+   wrk.body = readData("scripts/data.txt")
+   wrk.headers["Content-Type"] = "text/plain"
 
    if args[1] ~= nil then
       pipelineDepth = args[1]
    end
 
    local r = {}
    for i = 1, pipelineDepth, 1 do
-      r[i] = wrk.format(nil)
+      r[i] = wrk.format()
    end
 
    print("Pipeline depth: " .. pipelineDepth)
```

Comment:
@mikeharder Thoughts?

We have to either remove it altogether or leave it there. There's no hook in the scripts that allows us to do something per wrk run - it's always per thread.